src/pages/model/stackview/ModelPageStackViewContent.svelte

```diff
@@ -38,33 +39,41 @@
 
 <h1 class="sr-only">{$t('content.index.toolProcessCards.model.title')}</h1>
 <div class="flex flex-col h-full bg-backgrounddark">
-  <span class="sr-only" aria-live="polite"
-    >{$t('content.model.output.estimatedGesture.label', {
-      values: {
-        action: prediction.name,
-      },
-    })}</span>
-  <div class="flex justify-center space-x-10 py-5 text-xl border-b-3 border-b-gray-200">
-    <Information
-      underlineIconText={false}
-      isLightTheme={false}
-      iconText={$t('content.model.output.estimatedGesture.iconTitle')}
-      titleText={$t('content.model.output.estimatedGesture.descriptionTitle')}
-      bodyText={$t('content.model.output.estimatedGesture.descriptionBody')} />
-    <p class="font-semibold text-2xl">
-      {prediction.name}
-    </p>
-    {#if prediction.certainty}
-      <p class="bg-secondary text-white rounded w-15 text-center">
+  <GestureTilePart elevated class="flex mx-10 my-5 px-6 flex-row">
+    <span class="sr-only" aria-live="polite">
+      {$t('content.model.output.estimatedGesture.label', {
+        values: {
+          action: prediction.name,
+        },
+      })}
+    </span>
+    <div class="flex flex-grow justify-start gap-5 py-5 font-semibold text-2xl">
+      <p class="text-infoicondark">
+        {$t('content.model.output.estimatedGesture.iconTitle')}
+      </p>
+      <p>
+        {prediction.name}
+      </p>
+    </div>
+    <div class="flex flex-row justify-end m-auto gap-5">
+      <p
+        class="text-xl bg-secondary text-white rounded w-15 text-center"
+        class:invisible={!prediction.certainty}>
```

Comment:
I think I'd rather this be a conditional render as we don't need to save space. The info icon is naturally at the end of the flex container either way.